common/src/test/java/com/microsoft/identity/common/internal/cache/ActiveBrokerCacheTest.kt

```diff
@@ -35,42 +33,22 @@ import java.util.concurrent.TimeUnit
 
 class ActiveBrokerCacheTest {
```

Comment:
rename file/class

Reply:
There are test from the client cache here as well... although i can extract those out.